x/emissions/keeper/genesis.go

```diff
@@ -558,16 +558,6 @@ func (k *Keeper) InitGenesis(ctx context.Context, data *types.GenesisState) erro
 			}
 		}
 	}
-	//LatestOneInForecasterSelfNetworkRegrets []*TopicIdActorIdTimeStampedValue
```

Comment:
so we're deleting these lines from the genesis export, ok

I saw that there were some new additions to reputer.proto - shouldn't we be adding those in, here?